e2e-tests/tfm/cypress.config.js

```diff
@@ -28,12 +34,27 @@ module.exports = defineConfig({
     runMode: 2,
     openMode: 0,
   },
+  jwtSigningKey: process.env.JWT_SIGNING_KEY,
+  cookieSigningKey: process.env.SESSION_SECRET,
+  azureSsoAuthority: 'https://login.microsoftonline.com/',
```

Comment:
This could be ``${process.env.AZURE_SSO_AUTHORITY}/``

Reply:
Changed to use env var